### PR TITLE
Use ALCS document Source mapping

### PR DESCRIPTION
**Pull request by lstod**
-added enum to covert oats doc source to ALCS

### CODE REVIEW

File: bin/migrate-oats-data/documents/document_source_update.py [hunk 1]
```diff
@@ -124,6 +125,9 @@ def _get_document_source(data):
     alcs_source = data.get("source", "")
     if alcs_source != 'oats_etl' and alcs_source:
         return alcs_source
-    # Source is ALC/APP/LG as per 1558
-    else:
-        return source
\ No newline at end of file
+    elif source:
+        source = str(
+                OatsToAlcsDocumentSourceCode[source].value
```
Comment (mhuseinov) [inline bin/migrate-oats-data/documents/document_source_update.py:130]: Is it possible that the source is not in the oats enum, or are there only those 3 values in the documents table?

Comment (lstod) [reply bin/migrate-oats-data/documents/document_source_update.py:130]: oats_document_source_codes table only has 3 entries and the column is not null-able in the documents table

**Review by mhuseinov — APPROVED** [review]
👍

**Comment by urmikgov** [comment]
approved

**Pull request merged**